tools/structured-opt/structured-opt.cpp

```diff
@@ -41,6 +42,14 @@ static void registerIteratorDialects(DialectRegistry &registry) {
       >();
 }
 
+static void registerSubstraitDialects(DialectRegistry &registry) {
+  registry.insert<
+      // clang-format off
```

Comment:
TBH, I don't think this is worth keeping the format.

Reply:
Done. I think it is good to have this for lists that span multiple lines to make git diffs more readable. I started this format in case the list becomes multiple lines but, admittedly, that may never happen. Let me reintroduce the format suppression if and when I introduce a line break.